thoth/common/logging.py

```diff
@@ -149,6 +150,28 @@ def _get_sentry_integrations() -> List[object]:
     return integrations
 
 
+def before_send_handler(event, hint):
+    """Filter the errors caught before sending to Sentry.
+
+    This function ignores the exceptions passed in as a environment variable in a comma separated manner.
+    """
+    if len(_IGNORED_EXCEPTIONS) == 0:
+        return event
+    if 'exc_info' in hint:
+        exc_type, exc_value, tb = hint['exc_info']
+        for exception in _IGNORED_EXCEPTIONS:
+            module, name = exception
+            if module == getattr(exc_type, "__module__") and name == exc_type.__name__:
+                return None
+    elif 'log_record' in hint:
+        log_record = hint['log_record'].__dict__
```

Comment:
Why `__dict__` here? You can directly access the property:

```python
hint['log_record'].name
```

Reply:
Makes sense.